src/providers/oauth-token-provider.js

```diff
@@ -63,35 +63,44 @@ function OAuthTokenProvider() {
        */
 
       getAccessToken() {
-        return this.getToken() ? this.getToken().access_token : undefined;
+        const token = this.getToken();
```

Comment:
Sentry was picking up on errors like `Cannot read property 'access_token' of undefined`. Since we are using a ternary operator we are validating that the result of `getToken()` is not `undefined` however, when we are requesting the `access_token` property we are calling `getToken()` a second time and the access to the `access_token` property is failing due to the object being undefined.
The only valid explanation for this that I can think of, is that in the first call to `getToken()` an object was returned and in the second call nothing was returned, causing the error. As such, if we assign the value of `getToken()` to a `const` there won't be any change between the tested value and the value being accessed.